projects/kit/components/input-number/input-number.component.ts

```diff
@@ -70,11 +70,11 @@ export class TuiInputNumberComponent
 
     @Input()
     @tuiDefaultProp()
-    min = -Infinity;
+    min = Number.MIN_SAFE_INTEGER;
```

Comment:
What do you think, will you be able to add a unit test or a demo example with set max value inside for check on e2e?

Reply:
sure, will do that.